src/plugins/blocksmith/utils/setBlocks.js

```diff
@@ -0,0 +1,19 @@
+import dirtyEditor from '@triniti/cms/plugins/blocksmith/actions/dirtyEditor';
+import storeEditor from '@triniti/cms/plugins/blocksmith/actions/storeEditor';
+import convertToEditorState from '@triniti/cms/plugins/blocksmith/utils/convertToEditorState';
+import Message from '@gdbots/pbj/Message';
+
+export default (dispatch, formName, blocks, isDirty = false) => {
```

Comment:
@rsumilang why is there isDirty? looks like it is only being used like `setBlocks(dispatch, formName, formValue, true);`

Reply:
I wrote it like this because there is somewhere in the block editor that is also setting blocks but not dirtying it when loading the contents. It can possibly use this.